PWGMM/Rivet/Tools/README.md

```diff
@@ -0,0 +1,6 @@
+# Some tools
+
+- [`hepmc.py`](hepmc.py) Draw HepMC formatted data as a tree, one plot
+  per event.
```

Comment:
The documentation of the code _is_ already in the code at appropriate places.   The `README.md` files are meant to give a quick overview so that people know where to look.    In fact, at lot of ALICE code could do with a clear `README.md`